core/src/main/java/novoda/lib/sqliteprovider/provider/SQLiteContentProviderImpl.java

```diff
@@ -96,14 +95,12 @@ private Uri insertSilently(Uri uri, ContentValues values) {
     protected int updateInTransaction(Uri uri, ContentValues values, String selection, String[] selectionArgs) {
         ContentValues insertValues = (values != null) ? new ContentValues(values) : new ContentValues();
```

Comment:
😱 weird. the docs state that `null` is a possible value but...
<img width="519" alt="screen shot 2016-07-01 at 10 15 41" src="https://cloud.githubusercontent.com/assets/212528/16517363/e3c99c2e-3f74-11e6-84e5-19e727cce180.png">
oh well...